cmd/main.go

```diff
@@ -136,3 +152,31 @@ func main() {
 		os.Exit(1)
 	}
 }
+
+func initConfigMapTemplates(ctx context.Context, c client.Client, log logr.Logger) error {
```

Comment:
Consider renaming to initAssistedInstallerConfigMapTemplates

Reply:
Subsequent to this PR, I will be creating another one for IBI templates. This function will be used to apply those templates as well. Hence, the generic name.